packages/editor/src/components/block-list/block.js

```diff
@@ -86,6 +88,12 @@ export class BlockListBlock extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		if ( this.isForcingContextualToolbar && ! this.props.isTypingWithinBlock ) {
```

Comment:
Do we even need this condition? Or can it be assumed that since the render will occur following the assignment of the instance property that it's safe to immediately unset it in `componentDidUpdate`?

What is it about moving focus to the toolbar that causes `this.props.isTypingWithinBlock` to become `false`? It doesn't seem like something which could be strictly depended upon.

Reply:
> Do we even need this condition?
We don't need this condition and the code was updated. It is safe to immediately unset it, so the code was simplified.

> What is it about moving focus to the toolbar that causes this.props.isTypingWithinBlock to become false?

The expectation I had was that if the RichText loses focus, a stop typing action is dispatched.